tms/clickup/views.py

```diff
@@ -57,6 +54,7 @@ def update(self, request, *args, **kwargs):
 class ProjectDeleteView(DestroyAPIView):
     queryset = Project.objects.all()
```

Comment:
sorry i forgot to comment ,
here you create 2 class 1 for retrive and update and another for delete instend of that you can use this view
 RetrieveUpdateDestroyAPIView: